tests/rancher-cis-benchmark/rancher_cis_benchmark.go

```diff
@@ -0,0 +1,697 @@
+	FoundKey                     = "found"
+
+	PatchSaJobExistsCheck = "PatchSaJobExistsCheck"
+)
+
+var suite = test.Suite{
+	ChartPath: ChartPath,
+
+	Cases: []test.Case{
+		{
+			Name: "Using Defaults",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace),
+		},
+		{
+			Name: "Set Values.global.cattle.systemDefaultRegistry",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				SetValue("global.cattle.systemDefaultRegistry", "test-registry"),
+		},
+		{
+			Name: "Set Values.tolerations",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				Set("tolerations", testTolerations),
+		},
+		{
+			Name: "Set Values.nodeSelector",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				Set("nodeSelector", testNodeSelector),
+		},
+		{
+			Name: "Set .Values.global.cattle.psp.enabled to true",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				Set("global.cattle.psp.enabled", true),
+		},
+		{
+			Name: "Set .Values.global.cattle.psp.enabled to false",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				Set("global.cattle.psp.enabled", false),
+		},
+		{
+			Name: "Set Values.global.cattle.clusterName",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				SetValue("global.cattle.clusterName", "test-cluster"),
+		},
+		{
+			Name: "Set .Values.global.kubectl.repository and .Values.global.kubectl.tag",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				SetValue("global.kubectl.repository", "test-kubectl-repo").
+				SetValue("global.kubectl.tag", "v1.20.11"),
+		},
+		{
+			Name: "Set .Values.global.kubectl.imagePullPolicy to IfNotPresent",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				SetValue("global.imagePullPolicy", "IfNotPresent"),
+		},
+		{
+			Name: "Set .Values.global.kubectl.imagePullPolicy to Always",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				SetValue("global.imagePullPolicy", "Always"),
+		},
+		{
+			Name: "Set .Values.global.kubectl.imagePullPolicy to Never",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				SetValue("global.imagePullPolicy", "Never"),
+		},
+		{
+			Name: "Set .Values.image.cisoperator.repository and .Values.image.cisoperator.tag",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				SetValue("image.cisoperator.repository", "test/cis-operator").
+				SetValue("image.cisoperator.tag", "v1.0.0"),
+		},
+		{
+			Name: "Set securityScan and sonobuoy image values",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				SetValue("image.securityScan.repository", "test/security-scan").
```

Comment:
This can become one .Set instead of four SetValues
```
Set("image", map[string]string {
    "securityScan.repository": "test/security-scan",
    ...
}),
```

Reply:
done